aten/CMakeLists.txt

```diff
@@ -1,14 +1,17 @@
-if (BUILD_ATEN_MOBILE)
+if (BUILD_ATEN_MOBILE AND NOT BUILD_TORCH_MOBILE)
   return()
 endif()
 
 # Find modules
 list(APPEND CMAKE_MODULE_PATH
-  /usr/lib/x86_64-linux-gnu/
   ${CMAKE_CURRENT_SOURCE_DIR}/../cmake/Modules
   ${CMAKE_CURRENT_SOURCE_DIR}/../cmake/public
   ${CMAKE_CURRENT_SOURCE_DIR}/../cmake/Modules_CUDA_fix)
-list(APPEND CMAKE_LIBRARY_PATH /usr/lib/x86_64-linux-gnu/ /usr/lib/aarch64-linux-gnu/)
+
+if (NOT BUILD_TORCH_MOBILE)
+  list(APPEND CMAKE_MODULE_PATH  /usr/lib/x86_64-linux-gnu/)
+  list(APPEND CMAKE_LIBRARY_PATH /usr/lib/x86_64-linux-gnu/ /usr/lib/aarch64-linux-gnu/)
+endif()
```

Comment:
This changes lookup order. Hopefully it doesn't matter.

Reply:
I checked the history and given that @orionr explicitly changed the lookup order in #7163, it'd probably be more prudent to preserve it.

Reply:
I moved this now.